main.py

```diff
@@ -16,8 +18,44 @@ def create_pg_surface_from_color_and_size(color, size):
     return surf
 
 
+def create_overlays():
```

Comment:
Maybe this function could be moved into a different file to make main.py more readable / more fokused.